README.md

```
@@ -1,62 +1,120 @@
+   - **Sigil: Developer Settings: Api Url** – your API base URL (e.g. `http://localhost:80` or `http://localhost:5000`).
+3. Open the Sigil panel from the sidebar and use it; requests will go to your local API.
 
-This extension provides the following settings:
+### Project structure
 
-- `sigil.persona`: The persona that Sigil will embody in its responses (default: "Default")
-- `sigil.personalizeResponses`: Enable personalized responses based on feedback
-- `sigil.personalizedPrompt`: Custom preferences for tailoring Sigil's responses
-- `sigil.fieldStudyOptIn`: Consent to anonymous use of interactions in research publications
+| Path | Description |
+|------|-------------|
```

markdownlint-cli2 (0.20.0)

Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)